### DISCUSSION TITLE
2022.108.0 release is using old framework version

### DISCUSSION THREAD

**Discussion by huupoke** [ppy/osu#16361] [Q&A] [answered]
This release is using `2021.1227.0` version of the framework, which don't have the BASS-related changes. Is this intentional?

**Comment by bdach** [comment, marked as answer]
the bump is pending merge at https://github.com/ppy/osu/pull/16309. there was [one bass-related change](https://github.com/ppy/osu-framework/pull/4981) in the latest framework release but it should have exactly zero effect on anything as it was purely cosmetic.
